app/src/main/kotlin/com/intive/picover/main/theme/ColoredStatusBar.kt

```diff
@@ -0,0 +1,14 @@
+package com.intive.picover.main.theme
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.SideEffect
+import androidx.compose.ui.graphics.Color
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+
+@Composable
+fun ColoredStatusBar(color: Color) {
```

Comment:
The previous name was "against" composables naming rules but this one is misleading. It suggests that it is a status bar component but in reality, it only executes the side effect. Imo it should not be extracted to separate function but just used directly in `Theme.kt`

Reply:
True, probably it won't be reused anywhere